<launch>
    <node pkg="hrl_table_detect" name="tabletop_segmentor" type="tabletop_segmentor" respawn="true" output="screen">
        <param name="run_service" type="bool" value="true" />
        <param name="scan_width" value="1.5" />
        <param name="min_scan_depth" value="0.3" />
        <param name="max_scan_depth" value="3.0" />
        <param name="min_table_height" value="0.5" />
        <param name="max_table_height" value="1.0" />
        <param name="height_image_res" value="450" />
        <param name="inlier_magnitude" value="200" />
        <param name="num_edge_dilate" value="1" />
        <param name="num_closes" value="3" />
        <param name="degree_bins" value="1.0" />
        <param name="hough_thresh" value="0.20" />
        <param name="theta_gran" value="3.0" />
        <param name="rho_gran" value="0.1" />
        <param name="xgran" value="0.1" />
        <param name="ygran" value="0.1" />
        <param name="obj_min_area" value="0.01" />
    </node>

</launch>
